contracts/InterestRate/StableRate.sol

```diff
@@ -0,0 +1,248 @@
+     * @param _jumpMultiplierPerBlockForStable The multiplierPerBlock after hitting a specified utilization point
+     * @param _kink The utilization point at which the jump multiplier is applied
+     * @param _stableRatePremium The multiplierPerBlock after hitting a specified utilization point
+     * @param _optimalStableLoanRate Optimal stable loan rate percentage.
+     */
+    constructor(
+        uint256 _baseRatePerBlockForStable,
+        uint256 _multiplierPerBlock,
+        uint256 _multiplierPerBlockForStable,
+        uint256 _jumpMultiplierPerBlockForStable,
+        uint256 _kink,
+        uint256 _stableRatePremium,
+        uint256 _optimalStableLoanRate,
+        address owner_
+    ) {
+        owner = owner_;
+
+        updateStableRateModelInternal(
+            _baseRatePerBlockForStable,
+            _multiplierPerBlock,
+            _multiplierPerBlockForStable,
+            _jumpMultiplierPerBlockForStable,
+            _kink,
+            _stableRatePremium,
+            _optimalStableLoanRate
+        );
+    }
+
+    /**
+     * @notice Update the parameters of the interest rate model (only callable by owner, i.e. Timelock)
+     * @param _baseRatePerBlockForStable The approximate target base APR, as a mantissa (scaled by BASE)
+     * @param _multiplierPerBlock The rate of increase in interest rate wrt utilization (scaled by BASE)
+     * @param _multiplierPerBlockForStable The rate of increase in interest rate wrt utilization (scaled by BASE)
+     * @param _jumpMultiplierPerBlockForStable The multiplierPerBlock after hitting a specified utilization point
+     * @param _kink The utilization point at which the jump multiplier is applied
+     * @param _stableRatePremium The multiplierPerBlock after hitting a specified utilization point
+     * @param _optimalStableLoanRate Optimal stable loan rate percentage.
+     */
+    function updateStableRateModel(
+        uint256 _baseRatePerBlockForStable,
+        uint256 _multiplierPerBlock,
+        uint256 _multiplierPerBlockForStable,
+        uint256 _jumpMultiplierPerBlockForStable,
+        uint256 _kink,
+        uint256 _stableRatePremium,
+        uint256 _optimalStableLoanRate
+    ) external virtual {
+        require(msg.sender == owner, "StableRateModel: only owner may call this function.");
+
+        updateStableRateModelInternal(
+            _baseRatePerBlockForStable,
+            _multiplierPerBlock,
+            _multiplierPerBlockForStable,
+            _jumpMultiplierPerBlockForStable,
+            _kink,
+            _stableRatePremium,
+            _optimalStableLoanRate
+        );
+    }
+
+    /**
+     * @notice Calculates the utilization rate of the market: `borrows / (cash + borrows - reserves)`
+     * @param cash The amount of cash in the market
+     * @param borrows The amount of borrows in the market
+     * @param reserves The amount of reserves in the market (currently unused)
```

Comment:
```suggestion
     * @param reserves The amount of reserves in the market
```
Used on line 149

Reply:
Fixed, can check the changes at [this commit](2981e0d08b247b7c7a7dc3461e9701d7d593c520).